android/res/xml/backup_content_v31.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<data-extraction-rules>
+  <cloud-backup>
+    <include domain="file" path="bookmarks/"/>
+    <include domain="file" path="settings.ini"/>
+  </cloud-backup>
+  <device-transfer>
+    <include domain="file" path="bookmarks/"/>
+    <include domain="file" path="settings.ini"/>
+  </device-transfer>
```

Comment:
> This separation allows you, for example, to exclude a file or directory from Google Drive backups while still transferring it during device-to-device (D2D) transfers. This may be useful if you have files which would be too large to back up to the cloud, but which can be transferred between devices without issue.

I assume D2D transfers are for cases when a user switches to another device, e.g. an upgrade?
In this case we might consider adding map files there, otherwise the user will have to re-download all the maps **manually** and it could be a hassle.